debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MysqlDefaultValueAllZeroTimeIT.java

```diff
@@ -135,4 +135,60 @@ public void allZeroDateAndTimeTypeTest() throws InterruptedException {
         assertThat(schemaL.defaultValue()).isEqualTo(null);
 
     }
+
+    @Test
+    public void partZeroDateAndTimeTypeTest() throws InterruptedException {
+        config = DATABASE.defaultConfig()
+                .with(MySqlConnectorConfig.SNAPSHOT_MODE, MySqlConnectorConfig.SnapshotMode.INITIAL)
+                .with(MySqlConnectorConfig.TABLE_INCLUDE_LIST, DATABASE.qualifiedTableName("part_zero_date_and_time_table"))
+                .build();
+        start(MySqlConnector.class, config);
+
+        // Testing.Print.enable();
+
+        AbstractConnectorTest.SourceRecords records = consumeRecordsByTopic(9);
+        final SourceRecord record = records.recordsForTopic(DATABASE.topicForTable("part_zero_date_and_time_table")).get(0);
+
+        Schema schemaA = record.valueSchema().fields().get(1).schema().fields().get(0).schema();
+        Schema schemaB = record.valueSchema().fields().get(1).schema().fields().get(1).schema();
+        Schema schemaC = record.valueSchema().fields().get(1).schema().fields().get(2).schema();
+        Schema schemaD = record.valueSchema().fields().get(1).schema().fields().get(3).schema();
+        Schema schemaE = record.valueSchema().fields().get(1).schema().fields().get(4).schema();
+        Schema schemaF = record.valueSchema().fields().get(1).schema().fields().get(5).schema();
+        Schema schemaG = record.valueSchema().fields().get(1).schema().fields().get(6).schema();
+        Schema schemaH = record.valueSchema().fields().get(1).schema().fields().get(7).schema();
+
+        // column A, 0000-00-00 00:00:00 => 1970-01-01 00:00:00
+        ZonedDateTime a = ZonedDateTime.ofInstant(Instant.EPOCH, ZoneOffset.UTC);
+        String isoStringA = ZonedTimestamp.toIsoString(a, ZoneOffset.UTC, MySqlValueConverters::adjustTemporal);
+        assertThat(schemaA.defaultValue()).isEqualTo(isoStringA);
```

Comment:
```suggestion
        assertThat(schemaA.defaultValue()).isEqualTo("1970-01-01 00:00:00");
```

Can we just specify the expected value as-is? The less conversion/computation for expected values, the better.

Reply:
fix it